app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -131,6 +131,7 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
                 .createStatus(slug, pr.head.sha, statusRequest);
             lastUpdate.status = statusRequest.state;
             lastUpdate.head = pr.head.sha;
+            print('About to update: ${lastUpdate.pr}, ${lastUpdate.updates}');
```

Comment:
What's the reasoning for switching from using log statements to prints? IIRC AppEngine will still log stdout, but it doesn't assign a log level.

Reply:
Not intentional, just local testing, it ended up getting pushed. I'm considering this WIP until I can find out how I broke a test. :confused: